log/log.go

```diff
@@ -109,31 +119,49 @@ func writeErrorToFile(msg string) {
 	go errorLog.WriteString(fmt.Sprintf("[%s] %s\n", time.Now().Format("2006/0102/150405"), msg))
 }
 
+// checkInvalidChars checks if the formatted string contains '�' (U+FFFD)
+// and panics if env LOG_PANIC_ON_INVALIDCHARS bool is true.
+//
+// In production (LOG_PANIC_ON_INVALIDCHARS != true), this function returns immediately, no performance hit
+func checkInvalidChars(args ...interface{}) {
+	if panicOnInvalidChars {
+		s := fmt.Sprint(args...)
+		if strings.ContainsRune(s, '�') {
```

Comment:
@altergui seems that Codeac is failing here because it does not recognize this character: `�` and interprets it as `'''`

Reply:
haha does codeac come from prehistoric asciiland? that's a perfectly valid UTF-8 char. but ok, i'll escape it , thanks a million for digging into this (clicking on the codeac link prompted me for a login and i stopped there)

Reply:
amended in https://github.com/vocdoni/vocdoni-node/pull/590/commits/637166ea7d806eccbf9ec199dedea4d36b250f4e